test/standalone/test-debuglet.js

```diff
@@ -145,7 +145,17 @@ describe(__filename, function(){
     debuglet.start();
   });
 
-  it('should error if a package.json doesn\'t exist');
+  it('should error if a package.json doesn\'t exist', function(done) {
+    var config = extend({}, defaultConfig, {projectId: 'fake-project', workingDirectory: __dirname});
+    debuglet = new Debuglet(fakeDebug, config, logger);
+
+    debuglet.once('initError', function(err) {
+      assert(err);
```

Comment:
Actually, sorry, I misread this. We want to fail if `err` is not an error. I do not see a corresponding `assert.error` call.